lib/ASTGen/Sources/ASTGen/Attrs.swift

```diff
@@ -0,0 +1,1132 @@
+        .inlinable,
+        .lexicalLifetimes,
+        .lldbDebuggerFunction,
+        .marker,
+        .moveOnly,
+        .noAllocation,
+        .noDerivative,
+        .noEagerMove,
+        .noImplicitCopy,
+        .noLocks,
+        .noMetadata,
+        .nonEphemeral,
+        .nonEscapable,
+        .nonObjC,
+        .nonOverride,
+        .nsApplicationMain,
+        .nsCopying,
+        .nsManaged,
+        .objCMembers,
+        .objCNonLazyRealization,
+        .preconcurrency,
+        .propertyWrapper,
+        .requiresStoredPropertyInits,
+        .resultBuilder,
+        .sendable,
+        .spiOnly,
+        .showInInterface,
+        .specializeExtension,
+        .testable,
+        .transparent,
+        .uiApplicationMain,
+        .unsafeInheritExecutor,
+        .unsafeNoObjCTaggedPointer,
+        .unsafeNonEscapableResult,
+        .usableFromInline,
+        .used,
+        .warnUnqualifiedAccess,
+        .weakLinked:
+
+        return self.generateSimpleDeclAttr(attribute: node, kind: attrKind)
+
+      // Modifers.
+      case .accessControl:
+        // TODO: Diagnose and generateAccessControl().
+        fatalError("unimplemented")
+      case .referenceOwnership:
+        // TODO: Diagnose.
+        return self.generateReferenceOwnershipAttr(attribute: node, attrName: attrName)?.asDeclAttribute
+      case .async,
+        .consuming,
+        .borrowing,
+        .actor,
+        .distributedActor,
+        .required,
+        .optional,
+        .lazy,
+        .dynamic,
+        .infix,
+        .prefix,
+        .postfix,
+        .legacyConsuming,
+        .mutating,
+        .nonMutating,
+        .convenience,
+        .override,
+        .indirect,
+        .final,
+        .resultDependsOnSelf,
+        .knownToBeLocal,
+        .compileTimeConst:
+
+        // generateSimpleDeclAttr will diagnose and fix-it to change it to modifiers.
+        return self.generateSimpleDeclAttr(attribute: node, kind: attrKind)
+
+      // 'RejectByParser', these attribute kind should not be parsed as built-in attributes.
+      case .rawDocComment,
+        .objCBridged,
+        .synthesizedProtocol,
+        .staticInitializeObjCMetadata,
+        .restatedObjCConformance,
+        .clangImporterSynthesizedType,
+        .forbidSerializingReference,
+        .custom,
+        .setterAccess,
+        .rethrows,
+        .reasync:
+        // TODO: Diagnose or fallback to custom attributes?
+        return nil
+
+      case .none:
+        // Fall back to CustomAttr.
+        break
+      }
+    }
+
+    return self.generateCustomAttr(attribute: node)?.asDeclAttribute
+  }
+
+  func generateAlignmentAttr(attribute node: AttributeSyntax) -> BridgedAlignmentAttr? {
+    guard
+      let arg = node.arguments?.as(TokenSyntax.self)
+    else {
+      print("Not a token")
+      // TODO: Diagnose.
+      return nil
+    }
+    let value: Int? = Int(String(syntaxText: arg.rawText))
```

Comment:
We currently also support hex integer literals for the alignment’s value and values starting with `0` are considered octal values, which makes `010` a power of two and thus valid argument to `@_alignment`. Not sure if we need to continue supporting this since `@_alignment` is underscored.

Reply:
Oh interesting. I will add `FIXME`.